aggregate/housing_production/hpd_housing_ny_affordable_housing.py

```diff
@@ -20,28 +22,29 @@
 
 def load_housing_ny():
     """load the HPD Housing NY Units by Building dataset"""
-    df = pd.read_csv(
-        ".library/hpd_hny_units_by_building.csv",
-        usecols=[
-            "project_id",
-            "project_name",
-            "project_start_date",
-            "project_completion_date",
-            "number",
-            "street",
-            "borough",
-            "latitude_(internal)",
-            "longitude_(internal)",
-            "building_completion_date",
-            "reporting_construction_type",
-            "extremely_low_income_units",
-            "very_low_income_units",
-            "low_income_units",
-            "moderate_income_units",
-            "middle_income_units",
-            "other_income_units",
-        ],
-    )
+    df = read_from_S3("hpd_hny_units_by_building",
+                      "housing_production",
+                      cols=[
```

Comment:
Can we return this in a function so we can hard-code it farther down in the file? We made this change consistently in the QAQC app and it made our files much more readable

Reply:
down the file as in which part?

Reply:
Like code up a get_cols function so this can be hard-coded below the import logic. Makes the code more readable